content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -616,19 +617,10 @@ Volume binding matrix for statically provisioned volumes:
 
 | PV volumeMode | PVC volumeMode  | Result           |
```

Comment:
Can we simplify this table to just say volumeMode on PVC and PV must match?

Reply:
reduced to "`PersistentVolumeClaims` will only be bound to `PersistentVolumes` with the same `volumeMode.`" https://github.com/kubernetes/website/pull/10100/commits/c90a2a861cbe81ba9100a3fd20c16d91d6cb18eb#diff-bd23e5d028a7006d4c9e9cd8db0d5eabR614